apps/src/util/getScriptData.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Helper function to pull json data off of script tags.
+ *
+ * @param name string - The name of the data attribute to look up.
+ *     Note that this must be unique on the page.
+ * @returns the decoded json value of the data attribute.
+ *
+ * Example: given you have the following html:
+ *
+ *   <script src="myscript.js" data-someServerData='{"userId": "foobar"}'></script>
+ *
+ * you could use this function like so:
+ *
+ *   const userId = getScriptData('someServerData').userId;
+ *   console.log("the server that rendered this page claims the user is", userid);
+ *
+ */
+export default function getScriptData(name) {
+  name = name.toLowerCase();
+  const script = document.querySelector(`script[data-${name}]`);
+  try {
+    return JSON.parse(script.dataset[name]);
+  } catch (e) {
+    console.error("Failed to parse script data for script", name);
+    throw e;
+  }
+}
```

Comment:
There are other places we use this trick I believe (scriptOverview.js for example). Should we convert them to also use this?